packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart

```diff
@@ -8,7 +8,7 @@ import 'package:image_picker_platform_interface/image_picker_platform_interface.
 final String _kImagePickerInputsDomId = '__image_picker_web-file-input';
 final String _kAcceptImageMimeType = 'image/*';
 // TODO The value below seems to not be enough for Safari (https://github.com/flutter/flutter/issues/58532)
```

Comment:
Please remove this comment, it seems this change makes it obsolete!

Reply:
I removed the comment 👍🏻